modules/virt-aws-bm.adoc

```diff
@@ -25,14 +39,22 @@ For more information, see the {product-title} documentation about installing on
 Accessing virtual machines (VMs)::
 --
 * There is no change to how you access VMs by using the `virtctl` CLI tool or the {product-title} web console.
-* You can expose VMs by using a `NodePort` or `LoadBalancer` service. 
+* You can expose VMs by using a `NodePort` or `LoadBalancer` service.
 ** The load balancer approach is preferable because {product-title} automatically creates the load balancer in AWS and manages its lifecycle. A security group is also created for the load balancer, and you can use annotations to attach existing security groups. When you remove the service, {product-title} removes the load balancer and its associated resources.
 --
 
 Networking::
+// No SR-IOV in ROSA/OSD
+ifndef::openshift-rosa,openshift-dedicated[]
 --
 * You cannot use Single Root I/O Virtualization (SR-IOV) or bridge Container Network Interface (CNI) networks, including virtual LAN (VLAN). If your application requires a flat layer 2 network or control over the IP pool, consider using OVN-Kubernetes secondary overlay networks.
```

Comment:
I added some notes to remind myself why I made changes, should someone ask. Changed to: 
>Hiding the following in ROSA/OSD because SR-IOV is not supported in ROSA/OSD.